vkconfig_core/override.cpp

```diff
@@ -184,21 +187,21 @@ bool WriteSettingsOverride(const Environment& environment, const std::vector<Lay
         if (parameter.state != LAYER_STATE_OVERRIDDEN) continue;
 
         stream << "\n";
-        stream << "# " << layer->key.c_str() << "\n";
+        stream << "# " << layer->key.c_str() << "\n\n";
 
         std::string lc_layer_name = GetLayerSettingPrefix(layer->key);
 
         for (std::size_t i = 0, m = parameter.settings.size(); i < m; ++i) {
             const SettingData* setting_data = parameter.settings[i];
 
-            // Skip missing settings
-            const SettingMeta* meta = FindSetting(layer->settings, setting_data->key.c_str());
-            if (meta == nullptr) {
+            // Skip groups - they aren't settings, so not relevant in this output
+            if (setting_data->type == SETTING_GROUP) {
                 continue;
             }
 
-            // Skip settings with dependence not met
-            if (!::CheckDependence(*meta, parameter.settings)) {
+            // Skip missing settings
```

Comment:
How can we get missing settings?  Would this warrant a warning to `stderr`?

Reply:
I don't know. This code was there before I made changes to it. The diff looks like I added it, but the check was merely moved. The skip missing settings path does not get executed for any of the existing layers. I'll suggest filing an issue on this.